### PR TITLE
Implement baseline in torch. Fix bug too

### PR DESCRIPTION
**Pull request by wichersn**

### CODE REVIEW

File: extractors/configs/hptuning_config_torch.yaml [hunk 1]
```diff
@@ -24,7 +28,7 @@ trainingInput:
         - "0.5"
         - "0.75"
 
-      - parameterName: gin_bindings=TorchExtractor.randomize_weights
+      - parameterName: gin_bindings=torch_extractor.agent_model.randomize_weights
         type: CATEGORICAL
         categoricalValues:
         - "True"
```
Comment (ricvolpe) [inline extractors/configs/hptuning_config_torch.yaml:34]: Should we add `gin_bindings=TorchExtractor.weight_decay` to this scrip too?

File: extractors/torch_extractor.py [hunk 2]
```diff
@@ -119,21 +119,133 @@ def forward(self, x):
...
+        conv_layers.append(nn.Dropout(p=drop_rate))
+        input_size = layer_size
+    conv_layers.append(nn.Flatten())
+
+    conv_model = nn.Sequential(*conv_layers)
+    conv_output_shape = conv_model(torch.randn([1] + input_shape)).shape
+
+    fc_layer_sizes = extractor.get_layer_sizes(fc_first_size, fc_last_size, fc_num_layers)
+
+    layers = [conv_model]
```
Comment (ricvolpe) [inline extractors/torch_extractor.py:147]: Why do you build two models and then combine them as opposed to simply append all layers to the same list and then creating a sequential model (e.g. here you could assign `layers = conv_layers`)?

**Review by ricvolpe — APPROVED** [review]
Looks good to me

File: extractors/torch_extractor.py (hunk 2 above)
Comment (wichersn) [reply extractors/torch_extractor.py:147]: I need the conv_model so I can call it on a dummy input on line 143 to get the output shape.

File: extractors/configs/hptuning_config_torch.yaml (hunk 1 above)
Comment (wichersn) [reply extractors/configs/hptuning_config_torch.yaml:34]: Sure, but I'll let you add it since you already have it in your branch.

**Pull request merged**